### PR TITLE
Add missing documentation

### PR DESCRIPTION
**Pull request by mx-psi**
**Description:** 

- Enable `EXC0007` and `EXC0002` for non-internal, non-testbed packages.
- Add missing documentation.
- Add `nosec` directive to testbed.

**Link to tracking Issue:** Updates #2789

To be tracked in #2819.

### CODE REVIEW

File: config/parser.go [hunk 1]
```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// package config implements the configuration Parser.
+// Package config implements the configuration Parser.
```
Comment (bogdandrutu) [inline config/parser.go:15]: This comment is no longer correct.

Comment (mx-psi) [reply config/parser.go]: Removed (config package is already documented)

**Comment by mx-psi** [comment]
@bogdandrutu PTAL

**Review by bogdandrutu — APPROVED** [review]

**Comment by bogdandrutu** [comment]
@mx-psi I think you should remove for the moment the "enable" checks for the moment and we can merge this PR without that. Then we can try one more time to enable the checks, the project is moving too fast :)

**Comment by bogdandrutu** [comment]
@mx-psi I added a new commit to you PR to disable the 02 check. Can you please try one more time to enable that :)

**Comment by mx-psi** [comment]
@bogdandrutu makes sense to split into two steps given the activity (thanks for doing it yourself!), I will do it like that for future linters

**Pull request merged**